bodhi/server/scripts/approve_testing.py

```diff
@@ -64,7 +65,12 @@ def main(argv=sys.argv):
     if len(argv) != 2:
         usage(argv)
 
-    settings = get_appsettings(argv[1])
+    config_uri = argv[1]
+
+    setup_logging(config_uri)
+    log = logging.getLogger(__name__)
+
+    settings = get_appsettings(config_uri)
```

Comment:
I don't think we should change where the logs go with this patch - the messages should continue to go to the console. 

Something along these lines would work:

https://github.com/fedora-infra/bodhi/pull/2651/files#diff-70dadc71359f6d15c0061f0626e61917R34

In general, I don't think we need to stop using the ```print``` statement in here, we just want to set up a logger so the warning goes away.